libraries/AC_CustomControl/AC_CustomControl_PID.cpp

```diff
@@ -289,9 +290,9 @@ Vector3f AC_CustomControl_PID::update()
     // run rate controller
     Vector3f gyro_latest = _ahrs->get_gyro_latest();
     Vector3f motor_out;
-    motor_out.x = _pid_atti_rate_roll.update_all(target_rate[0], gyro_latest[0], 1);
-    motor_out.y = _pid_atti_rate_pitch.update_all(target_rate[1], gyro_latest[1], 1);
-    motor_out.z = _pid_atti_rate_yaw.update_all(target_rate[2], gyro_latest[2], 1);
+    motor_out.x = _pid_atti_rate_roll.update_all(target_rate[0], gyro_latest[0], _dt, true);
```

Comment:
IIUC _dt is correct here, but "limit" should really be false as we're not hitting limits.

Reply:
Thank you very much! I will fix it as soon as possible.